torchvision/prototype/transforms/_auto_augment.py

```diff
@@ -480,9 +503,10 @@ def __init__(
         chain_depth: int = -1,
         alpha: float = 1.0,
         all_ops: bool = True,
-        **kwargs: Any,
+        interpolation: InterpolationMode = InterpolationMode.NEAREST,
```

Comment:
Augmix uses bilinear:
```suggestion
        interpolation: InterpolationMode = InterpolationMode.BILINEAR,
```